### DISCUSSION TITLE
Merged Analysis of Multiomic data

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#959] [Questions / Documentation] [answered]
<img width="908" alt="Screen Shot 2021-08-05 at 10 38 23 AM" src="https://user-images.githubusercontent.com/64283689/128379100-a1c4a98f-fe12-4698-bb38-675b619af7db.png">
Is it possible to run a merged analysis of the Multiomic Datasets?

If yes, how do I generate seRNA from multiple datasets (in my case 2 datasets of different conditions)?
[ArchR-addGeneExpressionMatrix-5a9a7ad27e2f-Date-2021-08-05_Time-10-33-29.log](https://github.com/GreenleafLab/ArchR/files/6939907/ArchR-addGeneExpressionMatrix-5a9a7ad27e2f-Date-2021-08-05_Time-10-33-29.log)

Need help with reading in the scRNA data for two separate multiome datasets being processed together.

Thanks!

**Comment by sylestiel** [comment]
Still awaiting some suggestions on how to read in the scRNA data when merging/integrating multiomic data.

**Comment by rcorces** [comment, marked as answer]
Response times are going to be slower for the next week since I am the only one currently working on ArchR and I'm on vacation.
This issue was previously solved
https://github.com/GreenleafLab/ArchR/issues/507#issuecomment-780709104

**Comment by sylestiel** [comment]
My apologies!!
